lib/src/calendar_event_data.dart

```diff
@@ -96,4 +96,156 @@ class CalendarEventData<T extends Object?> {
 
   @override
   int get hashCode => super.hashCode;
+
```

Comment:
So this is the best way I could come up with to find the CalendarEventData when the start/end time keeps changing.